scrapinghub/client/activity.py

```diff
@@ -10,51 +10,51 @@ class Activity(_Proxy):
     Not a public constructor: use :class:`Project` instance to get a
     :class:`Activity` instance. See :attr:`Project.activity` attribute.
 
-    Please note that list() method can use a lot of memory and for a large
```

Comment:
The problem with moving this to a parent class is that it becomes invisible when you check help or docs for the method. E.g. I think we should build documentation based on docstrings later on, this line should be in each list method that can cause such issues.